README.md

```diff
@@ -27,8 +27,8 @@
 - [Rspack GitHub Repo](https://github.com/web-infra-dev/rspack)
 - [Rspack Release Notes](https://github.com/web-infra-dev/rspack/releases)
 - [Rspack Examples](https://github.com/rspack-contrib/rspack-examples): Examples for Rspack, Rsbuild, Rspress and Rsdoctor.
-- [Rsfamily Design Resources](https://github.com/rspack-contrib/rsfamily-desgin-resources): 
-Design resources for Rspack, Rsbuild, Rspress and Rsdoctor.
+- [Rsfamily Design Resources](https://github.com/rspack-contrib/rsfamily-desgin-resources):
+  Design resources for Rspack, Rsbuild, Rspress and Rsdoctor.
```

Comment:
Is this a mistake?

Reply:
![image](https://github.com/web-infra-dev/awesome-rspack/assets/165243096/f62e4b27-e7f1-4640-ac3b-017b94c8e9d0)

Reply:
Sorry, my negligence led to an unnecessary process. The issue has now been resolved by merging through commit.

Reply:
It looks like the change is still there, can you check again?